### ISSUE TITLE
stroke width tool not showing inside of visible screen

### ISSUE THREAD

**Issue by eamayreh** [pubpub-zz/ppInk#63]
Hi. Just noticed this as I moved the toolbox to use at the top area: 
stroke width not showing inside of screen (I thought button was not working first ;)

here's a quick demo; it's not a big problem though; right now I'm using it at bottom area which is fine.

https://user-images.githubusercontent.com/61389799/110225233-239ff800-7eec-11eb-976d-fd8e0729ed43.mp4

**Comment by pubpub-zz** [comment]
PenWidth panel should be fixed now

[ppInk8.zip](https://github.com/pubpub-zz/ppInk/files/6104676/ppInk8.zip)

Also includes fixes for #60,#62,#59

**Comment by eamayreh** [comment]
@pubpub-zz 

https://user-images.githubusercontent.com/61389799/110551974-2f372d00-813f-11eb-9e87-aa4be16c39ae.mp4

here are the video highlights (I hope it's not too lengthy):
- ctrl+enter is working fine and is very convenient, thank you : )
- shortcuts for other tools not working after selecting **and using** text tool (if at first you just select text tool **without using it**, you **can** switch to other tools via their respective shortcuts and use them normally)

- penwidth panel might still disappear if toolbox moved to very top of screen (maybe you can limit where it can be placed to prevent this?)
- noticed clipart window (when you want to insert a clipart) will show out of screen if toolbar is at top area

**Comment by pubpub-zz** [comment]
@eamayreh 
thanks for very good testing
number chip, panels position(penwidth and clipart dlg) are fixed. available inhere
[ppInk10a.zip](https://github.com/pubpub-zz/ppInk/files/6115878/ppInk10a.zip)
For me key shortcuts are working. can you try and make a video record with ppInk in Window mode (right click on snap button to define the window). 
Thx again for your tests

**Comment by eamayreh** [comment]
> 
> 
> @eamayreh
> thanks for very good testing
> number chip, panels position(penwidth and clipart dlg) are fixed. available inhere
> [ppInk10a.zip](https://github.com/pubpub-zz/ppInk/files/6115878/ppInk10a.zip)
> For me key shortcuts are working. can you try and make a video record with ppInk in Window mode (right click on snap button to define the window).
> Thx again for your tests

@pubpub-zz 
Haha, I think I found the culprit, I closed all my tray apps and began trying enabling one by one (finally :D , but at the same time not so good for me :(

The program that causes all of this is a free virtual desktop app called **Dexpot**. I use it to organize my desktops, so I can have more than one desktop screen, each with its own icons;  it's the only one that provides this simple yet very handy functionality.

Here's a video just to show that the same happens even in windowed mode while this app is running, despite disabling all of its hotkeys.

https://user-images.githubusercontent.com/61389799/110681043-061ba880-81e2-11eb-9fa1-209f3c770568.mp4



As I've mentioned above, I disabled all the app hotkeys, but unfortunately that didn't work. The only way it worked was to close it all together, only then did all the shortcuts in PPink work correctly.

At least we figured the problem is on my end, so we can close this issue as everything else seems to be working great. 
Thank you so much.

P.S. Is there any chance to include some option in PPink to allow it to force its hotkeys, or override other hotkeys set by other application ( as an option in the settings )? 🤓
